og/bump/action.yml

```diff
@@ -49,4 +49,5 @@ runs:
           ${{ inputs.pr_title }}
         EXPECTMENOTINTHYPRTITLE
         python -c "$(wget -qSO- https://raw.githubusercontent.com/XanaduAI/cloud-actions/${{ inputs.dev_branch }}/og/bump/tagger.py 2>/dev/null)" 
+        git add .;
```

Comment:
What will we add here? The newly minted changelog?

Reply:
Exactly! `git commit -a` will not add new files